galleries/examples/shapes_and_collections/ellipse_arrow.py

```diff
@@ -0,0 +1,108 @@
+"""
+===================================
+Ellipse with orientation arrow demo
+===================================
+
+This demo shows how to draw an ellipse with
+an orientation arrow (clockwise or counterclockwise).
+Compare this to the :doc:`Ellipse collection example
+</gallery/shapes_and_collections/ellipse_collection>`.
+"""
+
+import matplotlib.pyplot as plt
+from matplotlib.markers import MarkerStyle
+from matplotlib.patches import Ellipse
+from matplotlib.transforms import Affine2D
+
+
+# Create a figure and axis
+fig, ax = plt.subplots(1, 1, subplot_kw={"aspect": "equal"})
```

Comment:
```suggestion
fig, ax = plt.subplots(subplot_kw={"aspect": "equal"})
```
can be left out, because these are the defaults.